lib/at_cmd_parser/at_cmd_parser.c

```diff
@@ -31,6 +36,8 @@ enum at_parser_state {
 
 static enum at_parser_state state;
 
+static bool set_type_string = false;
```

Comment:
Checkpatch complains about initializing static bool with false (which will be implicitly initialized), lets make it (`static bool set_type_string`).

Reply:
Agreed. Fixed.